wilson/util/__init__.py

```diff
@@ -1 +1,38 @@
 """Utility modules for wilson."""
+
+from wilson.util.common import EFTutil as _EFTutil
```

Comment:
What's this for? To prevent importing `EFTutil` from `wilson.util`? But why not, actually?

Reply:
I agree, the underscore is not really needed. `EFTutil` can be imported from `wilson.util.common` already, but it might actually be useful if it can also be imported from `wilson.util` directly.